tests/misc_env/push_cosbench_workload.py

```diff
@@ -87,7 +88,7 @@
   <workflow>
     <workstage name="preparing_cluster">
         <work type="prepare" workers="1" config="cprefix=bucket_prefix;containers=r(1,bucket_count);
-        objects=r(1,objects_count);sizes=h(1|5|25,5|50|40,50|256|25,256|512|5,512|1024|3,1024|5120|1,5120|51200|1)KB"/>
+        objects=r(obj_num_start,objects_count);sizes=h(1|5|25,5|50|40,50|256|25,256|512|5,512|1024|3,1024|5120|1,5120|51200|1)KB"/>
```

Comment:
@anrao19 could you please replace "objects_count" with something similar to "obj_num_end"
and calculate obj_num_end = obj_num_start + objects_count in the below code, and replace obj_num_end in the workload conf

Reply:
@anrao19 , I don't think that will work.

Reply:
okay no problem @anrao19 
let me look into it in the coming PR..